bazel/envoy_build_system.bzl

```diff
@@ -43,7 +43,27 @@ def envoy_linkopts():
             # See note here: http://luajit.org/install.html
             "-pagezero_size 10000", "-image_base 100000000",
         ],
+        "//bazel:enable_exported_symbols": [
+            '-L/usr/local/lib/lua/5.1/',
+            "-pthread",
+            "-lrt",
+            "-ldl",
+            # Force MD5 hash in build. This is part of the workaround for
+            # https://github.com/bazelbuild/bazel/issues/2805. Bazel actually
+            # does this by itself prior to
+            # https://github.com/bazelbuild/bazel/commit/724706ba4836c3366fc85b40ed50ccf92f4c3882.
+            # Ironically, forcing it here so that in future releases we will
+            # have the same behavior. When everyone is using an updated version
+            # of Bazel, we can use linkopts to set the git SHA1 directly in the
+            # --build-id and avoid doing the following.
+            '-Wl,--build-id=md5',
+            '-Wl,--hash-style=gnu',
+            '-Wl,-E',
```

Comment:
Instead of replicating the entire block just to add `-E` could we possibly have this be the output of a function that keys off of whether this feature is enabled or not? Then we can just call the function below?